app/src/main/java/edu/uc/groupProject/topten/dto/TopTenList.kt

```diff
@@ -19,9 +19,11 @@ data class TopTenList(
     var id:Int, var title:String, var description:String="", var isActive:Boolean,
     var category:String /* Possible category dto? */, var publishDate:Date,
      var expireDate: Date
+
 ){
     private var mlistItems:List<ListItem> = ArrayList<ListItem>()
 
+
```

Comment:
what does "m" stand for in mlistItems? Just curious (the IDE thinks it's a typo lol).

Reply:
This is often a naming convention for private/internal variables.